### DISCUSSION TITLE
Old YouTube icon option

### DISCUSSION THREAD

**Discussion by randoomdude** [ReVanced/revanced-discussions#798] [General] [answered]
Like it says, would be nice for their to he an option to have the old red YouTube icon so it matches the YouTube music one

**Comment by SlyPredator** [comment]
Disable/exclude `custom-branding` patch then while building YT Revanced.

**Reply by LisoUseInAIKyrios** [reply, marked as answer]
The custom branding patch now allows setting a custom icon file using patch options.  The end user will need to provide the icon file.

**Comment by LisoUseInAIKyrios** [comment]
You can pick a
